src/pages/zh-cn/core-concepts/astro-components.md

```diff
@@ -75,9 +76,9 @@ const data = await fetch ('SOME_SECRET_API_URL/users').then (r => r.json ());
 
 如果你在这里写普通的 HTML，你的组件将在任何 Astro 页面上呈现它被导入和使用的 HTML。
 
-然而，Astro 的组件模板语法也支持 **JavaScript 表达式**、**导入的组件** 和 **[特殊的 Astro 指令](/zh-cn/reference/directives-reference/)**。在组件脚本中定义的数据和值（在页面构建时）可以在组件模板中使用，以产生动态创建的 HTML。
+然而，Astro 的组件模板语法也支持 **JavaScript 表达式**、**导入的组件**和**[特殊的 Astro 指令](/zh-cn/reference/directives-reference/)**。在组件脚本中定义的数据和值（在页面构建时）可以在组件模板中使用，以产生动态创建的 HTML。
```

Comment:
Looks like this change broke the bold formatting:

<img width="420" alt="" src="https://user-images.githubusercontent.com/357379/191047127-4b7df47c-b1f6-4015-ab90-e2e34a67f4c6.png">

Do we need a space like this?

```suggestion
然而，Astro 的组件模板语法也支持 **JavaScript 表达式**、**导入的组件**和 **[特殊的 Astro 指令](/zh-cn/reference/directives-reference/)**。在组件脚本中定义的数据和值（在页面构建时）可以在组件模板中使用，以产生动态创建的 HTML。
```

Reply:
Or if the space isn’t grammatically correct, we might be able to use HTML directly:

```suggestion
然而，Astro 的组件模板语法也支持 **JavaScript 表达式**、**导入的组件**和<strong>[特殊的 Astro 指令](/zh-cn/reference/directives-reference/)</strong>。在组件脚本中定义的数据和值（在页面构建时）可以在组件模板中使用，以产生动态创建的 HTML。
```